### ISSUE TITLE
TTTTimeIntervalFormatter idiomatic date two days ago sometimes referred to as "yesterday"

### ISSUE THREAD

**Issue by jnic** [FormatterKit/FormatterKit#152]
Reproduction steps:

```
TTTTimeIntervalFormatter *formatter = [[TTTTimeIntervalFormatter alloc] init];
formatter.usesIdiomaticDeicticExpressions = YES;
NSDate *date = // 2015-02-22 15:33:50 +0000
NSDate *now = // 2015-02-24 10:13:39 +0000
NSString *human = [formatter stringForTimeIntervalFromDate:now toDate:date];
NSLog(@"Now: %@, Date: %@, Human: %@", now, date, human);
```

Output:

```
Now: 2015-02-24 10:13:39 +0000, Date: 2015-02-22 15:33:50 +0000, Human: yesterday
```

Expected output:

```
Human: 2 days ago
```

This line in TTTTimeIntervalFormatter looks only at the "day" component:

```
} else if ([components day] == -1 && [components year] == 0 && [components month] == 0 && [components weekOfYear] == 0) {
        return @"yesterday";
}
```

This results in all dates that are more than 24 hours ago, but less than 48 hours ago, being referred to as "yesterday", even when midnight has passed twice.

**Comment by sebastianludwig** [comment]
I'm working on a fix. Give me a few days.

**Comment by damienrambout** [comment]
👍 
The same issue occurs with "tomorrow" (or "1 day from now") when there's clearly 2 **week days** separating two dates.

**Comment by mattt** [comment]
Fixed by #189.
